lib/resources/powerpacks.js

```diff
@@ -108,17 +114,42 @@ export class Powerpack extends PlivoResource {
 		let path = 'NumberPool/' + this.number_pool_id + '/Number/' + number + '/';
 		return super.customexecuteAction(path.toString().trim(), 'DELETE', params);
 	}
+	remove_tollfree(tollfree, unrent = false) {
+	var params = {};
+		if (typeof unrent === 'boolean') {
```

Comment:
you expecting unrent = false in function arg, why do you need type check for unrent?

Reply:
changed